python/nano/test/pytorch/tests/inference/test_ipex_jit_inference.py

```diff
@@ -103,6 +113,49 @@ def test_ipex_jit_inference(self):
         with InferenceOptimizer.get_context(new_model):
             new_model(self.data_sample)
     
+    def test_ipex_channels_last_inference(self):
```

Comment:
One suggestion here, may not change your code.
For this case, maybe just define one ut like `test_jit_ipex_channels_last_inference` here and list three cases inside the the function, so you don't need to define model, x1, x2, x3 three times.